buildpack.md

```diff
@@ -770,17 +777,21 @@ The stack `build-images` and `run-images` are suggested sources of images for pl
 [[processes]]
 type = "<process type>"
 command = "<command>"
+args = ["<arguments>"]
 
 [[slices]]
 paths = ["<app sub-path glob>"]
 ```
 
 The buildpack MAY specify any number of processes or slices.
 
-For each process, the buildpack MUST specify:
+For each process, the buildpack:
 
-- A unique process type for each entry within a `launch.toml` file.
-- A command that is valid when executed using the Bash 3+ shell.
+- MUST specify a `type` that is not identical to other process types provided by the same buildpack.
+- MUST specify a `command` that is either:
+  - A command sequence that is valid when executed using the Bash 3+ shell, if `args` is not specified.
```

Comment:
If Bash is available and `args` is specified, what is the expected behavior. Will the lifecycle invoke with `execve` anyways? Could that cause problems for authors who provide `args` without realizing it's meant for non-Bash?

Reply:
Given the comment earlier about `.profile.d` is seems it could trip people up

Reply:
I'm less worried in this case compared to `docker run` / `CMD`, but still definitely a concern.

Reply:
Oh I was expecting we'd use `command = "- ..."` here too. Hmmm

Reply:
It doesn't work well for `command`, because it's not a list. I don't feel quite as comfortable assuming that `"- ..."` has special meaning. `argv[1] == "-"` seems safer.

I did add a `direct` boolean to control this behavior for `launch.toml` -- not sure if that's clear by the highlighted line.

Reply:
If we used `-` in command, it would always need to be in the form:
```
- binary
```
or
```
- /path/to/binary
```
but
```
- binary arg1
```
would need to be special-cased to fail.

Reply:
Should we consider removing `args` and allowing `command` to be either a string or an array (where the array behaves like you describe with `args`)? Kinda resembles `CMD` this way, which might actually be bad if it has different implications. 

Also not sure how hard it is to handle a key that can be a string or an array in TOML parsers

Reply:
You'd probably have to manually serialize.

Reply:
1. Not a huge fan of complex schema
2. `command` is special in direct mode, e.g., we may run `LookPath` on it.
3. `command` is special in bash mode, e.g., you may actually want to pass args directly to a shell invocation.